go/tasks/pluginmachinery/core/resource_manager.go

```diff
@@ -33,9 +33,44 @@ type ResourceRegistrar interface {
 	RegisterResourceQuota(ctx context.Context, namespace ResourceNamespace, quota int) error
 }
 
-// Resource Manager manages a single resource type, and each allocation is of size one
+// Resource Manager manages type, and each allocation is of size one
```

Comment:
`Resource manager manages type` -> ResourceManager provides a task type specific pooling system for Flyte Tasks.
Plugin writers can optionally request for resources in their tasks, in single quantity. (Now explain how the usage should be)

Reply:
I guess i would clean up this documentation to follow a format like
```
1. Define terms, Resource, allocation token etc...
2. describe resource manager
3. Explain usage
4. Example (Qubole that you have)
```